apps/docs/pages/guide/create-commandkit.mdx

```diff
@@ -12,14 +14,28 @@ npm create commandkit@latest
 
 This will start the CLI in the current directory.
 
+### Development version
+
+Similar to the main package, you could try the `dev` or `next` versions of create-commandkit:
+
+<Callout type="warning">
+  This version is likely to have issues. Be careful!
+</Callout>
+
+```sh copy
+npm create commandkit@dev
+```
+
 ## Prompts
 
 When running create-commandkit, you should be asked the following questions:
 
-- **Project Directory:** Defines where your project will be located. It defaults to the current working directory.
+- **Directory:** Defines where your project will be located. It defaults to the current working directory.
 
 - **Package Manager:** Lets you choose which package manager to use — npm, pnpm, or Yarn.
 
+- **Language:** The language to use for your project — JavaScript or TypeScript.
```

Comment:
It might be better to point out over here that the language option is currently only available in the development version?